### DISCUSSION TITLE
Runner , job build matrix /statistics

### DISCUSSION THREAD

**Discussion by roshvin** [actions/actions-runner-controller#1686] [Questions]
we are running action runner at our GHE enterprise . Now management is asking some statistics 

is there any solution to get below information from ARC

1. number of runners provisioned .
2. number of jobs run 
3. repository used this runner .

**Comment by roshvin** [comment]
Hi @mumoshu could you pls provide any suggestion

**Comment by ViktorLindgren95** [comment]
https://github.com/cpanato/github_actions_exporter

There are multiple projects that export these metrics to prometheus
